CONTRIBUTING.md

```diff
@@ -6,17 +6,34 @@ The [list of issues](https://github.com/mimblewimble/grin/issues) is a good plac
 
 Additional tests are rewarded with an immense amount of positive karma.
 
-More documentation or updates/fixes to existing documentation are also very welcome. However, if submitting a PR consisting of documentation changes only, please try to ensure that the change is significantly more substantial than one or two lines. For example, working through an install document and making changes and updates throughout as you find issues is worth a PR. For typos and other small changes, either contact one of the developers, or if you think it's a significant enough error to cause problems for other users, please feel free to open an issue.
+More documentation or updates/fixes to existing documentation are also very welcome. However, if submitting a PR(Github Pull-Request) consisting of documentation changes only, please try to ensure that the change is significantly more substantial than one or two lines. For example, working through an install document and making changes and updates throughout as you find issues is worth a PR. For typos and other small changes, either contact one of the developers, or if you think it's a significant enough error to cause problems for other users, please feel free to open an issue.
 
-Find us:
+# Find Us
 
+If any help is needed during your effort to contribute on this project, please don't hesitate to contact us:
 * Chat: [Gitter](https://gitter.im/grin_community/Lobby).
+* [Forum](https://www.grin-forum.org/)
+* [Website](https://grin-tech.org)
 * Mailing list: join the [~MimbleWimble team](https://launchpad.net/~mimblewimble) and subscribe on Launchpad.
+* News: [Twitter](https://twitter.com/grinmw). Twitter bot that scrapes headlines, mailing list, and reddit posts related to MimbleWinble/Grin.
+
+# Pull-Request Name Prefix
```

Comment:
Was it discussed on Gitter? Not against it but I don't see why it belongs in this PR.

Reply:
Ah, just copy & paste.  Do you mean `News` is not official?

I'm touching this doc, see the different `Find us`, so, ...   don't want to wait 😺

Reply:
No I mean the prefixes. Sorry if that wasn't clear.

Reply:
Okay 😄   Yes, it belongs. Because in this PR, the `change log` will be generated from these `issues` and `pull-request`. We don't want to generate a mess change log 😸 

And that's why I said I request `serious` review and send invite message in Gitter to give review on this PR.

Reply:
I update and add the explicit [draft, to be reviewed and discussed] words there.  
and a forum post for this discussion is submitted on:  https://www.grin-forum.org/t/github-pr-pull-request-title-prefix-rule-optional/842  , so let's continue discussion there.